docs/gestor-dependencias.md

```diff
@@ -0,0 +1,41 @@
+# Gestor de dependencias
+
+## Criterios de búsqueda de un gestor de dependencias:
+
+Los gestores de dependencias son herramientas específicas para llevar a cabo la instalación de todas las necesarias de un proyecto en un lenguaje determinado. Para elegir uno para nuestra aplicación, se establecen los siguientes criterios:
+
+- Que sea un gestor de dependencias de JavaScript, a ser posible basado en NodeJS.
+- Que utilice el fichero de dependencias package.json (buenas prácticas).
+- No necesariamente debe funcionar muy bien para proyectos muy grandes, puesto que este no lo es.
+- Que tenga buena documentación.
+
+## Gestores de dependencias que cumplen los criterios:
+
+### npm
+
+(Node Package Manager) Como su nombre lo indica, es el gestor de paquetes de NodeJS, y permite instalar, actualizar y publicar las dependencias y paquetes tanto públicas como privadass de la comunidad de NodeJS. Nos permite gestionar tanto dependencias como tareas, aunque en menor medida (como hemos visto en #27).
+
+- A pesar de ser el más utilizado desde hace años, tiene dos principales problemas: es lento y el orden de instalación de las dependencias no es predecible, aunque estos no son realmente problemas importantes en proyectos pequeños, pero sí lo son cuando se manejan proyectos de gran tamaño entre muchas personas.
+- Tiene gran cantidad de documentación, ya que es el más estandarizado y el más utilizado por la comunidad.
+
+### Yarn
+
+Es un gestor dependencias de JavaScript, basado en npm, que está muy enfocado en la velocidad y la seguridad, muy rápido y muy fácil de usar. Además, devuelve un feedback al usuario bastante amigable. Las principales diferencias que tiene respecto a npm a tener en cuenta son:
```

Comment:
De veras, ¿por qué escribís esto? Si no fuera un gestor de dependencias de javascript, ¿qué hacía aquí? ¿Por qué ponéis el panfleto publicitario? ¿Eso ayuda a elegir a este y no otro? ¿Los otros no están enfocados a la velocidad?